pkgs/development/python-modules/stups-zign/fix_tests.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tests/test_api.py b/tests/test_api.py
+index 34b3679..2267a7e 100644
+--- a/tests/test_api.py
++++ b/tests/test_api.py
+@@ -31,7 +31,7 @@ def test_get_new_token_server_error(monkeypatch):
+     with pytest.raises(zign.api.ServerError) as excinfo:
+         zign.api.get_new_token('myrealm', ['myscope'], 'myuser', 'mypass', 'http://example.org')
+ 
+-    assert 'Server error: Token Service returned HTTP status 500' in str(excinfo)
++    assert 'Server error: Token Service returned HTTP status 500' in str(excinfo.value)
```

Comment:
this should be upstreamed. I would make a PR, "Enable pytest5 compatibility", then just use fetchpatch to then fetch your commit diff. Then we can remove the patch when they bump the version :)